### PR TITLE
Add pois indexes for polygon and point tables

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/tilezen/vector-datasource/issues/983

@zerebubuth could you review please?

### CODE REVIEW

File: data/migrations/v1.0.0-point.sql [hunk 1]
```diff
@@ -10,3 +10,39 @@ UPDATE
...
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_12_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_12_index RENAME TO planet_osm_point_min_zoom_pois_12_index;
+COMMIT;
+
+BEGIN;
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_15_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_15_index RENAME TO planet_osm_point_min_zoom_pois_15_index;
+COMMIT;
+
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_way_index;
```
Comment (zerebubuth) [inline data/migrations/v1.0.0-point.sql]: I think the existing index is called `planet_osm_point_min_zoom_wa9_index` for the `IS NOT NULL` constraint.

Comment (rmarianski) [inline data/migrations/v1.0.0-point.sql]: Good catch.

Comment (rmarianski) [inline data/migrations/v1.0.0-point.sql]: Updated in 58c85dc.

**Comment by zerebubuth** [comment]
:+1: looks good

**Pull request merged**